provision-enterprise_catalog.sh

```diff
@@ -0,0 +1,32 @@
+name="enterprise_catalog"
+port="18160"
+
+docker-compose up -d
```

Comment:
We only need to start catalog service for this

```suggestion
docker compose up -d $name
```